website_product_subscription/controllers/subscribe.py

```diff
@@ -0,0 +1,268 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Coop IT Easy SCRLfs <http://coopiteasy.be>
+#   Rémy Taymans <remy@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import http
+from openerp.exceptions import ValidationError
+from openerp.http import request
+from openerp.tools.translate import _
+
+from openerp.addons.website_product_subscription.controllers.subscribe_form import SubscribeForm
+
+
+class SubscribeController(http.Controller):
+
+    @http.route(
+        [
+            # '/page/become_subscriber',
+            # '/become_subscriber',
+            '/new/subscription',
```

Comment:
@remytms You should grep `become_subscriber`, there are still uses of this route in the codebase.

Reply:
@robinkeunen Yes, but I'm not sure of removing `/become_subscriber` page. For now, I prefer giving the new url to Médor. And keep the old one to keep compatibility with the old behaviour. Because, medor will perhaps change the default behavour of this form.

Reply:
Ok to keep the old one. Don't forget to communicate the new routes to Jonathan.